infrastructure/helm/bailo/templates/bailo/bailo.secret.yaml

```diff
@@ -6,6 +6,17 @@
 {{- $rootMinioUserValue = index $minioSecret.data "root-user" }}
 {{- $rootMinioPasswordValue = index $minioSecret.data "root-password" }}
 {{- end -}}
+{{- if not .Values.minio.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "bailo.fullname" . }}-minio
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  root-password: {{ .Values.minio.password | b64enc | quote }}
```

Comment:
Can we call these what they are- access key and secret key?

Reply:
unfortunately, minio helm chart from Bitnami looks for these vars.